apiary.apib

```diff
@@ -41,7 +41,9 @@ API keys are used as Authorization tokens in the header in all authenticated API
 SSO is also available via our Integrations and via SAML 2.0.
 
 # Rate Limiting
-We allow up to 200 requests every 30 seconds from a single IP address. If you attempt to make requests more quickly then this you may receive a `429 Too Many Requests` response from an endpoint.
+We allow up to 2000 requests every 5 minutes from a single IP address, but some higher security endpoints have stricter rate limiting. Requests are counted in 30 second periods.
```

Comment:
how to verify this 2000/5 RPM thing? I can see we have rules where rate limit is set to 100. 
<img width="450" alt="image" src="https://user-images.githubusercontent.com/12901870/230073913-cdc28112-b88f-4fab-92d1-6d216c27b2fc.png">

Reply:
So the least restrictive rule we have is the "Default Throttle" one, which is where the rate limit is 2000. This is the widest net, so to speak, and is the last rule to be matched to all requests.

We did this to stay as similar as possible to when we were using Rack Attack. At that time, the broadest rule/default throttle was 200 requests per 30 seconds, so 2000 per 300 seconds is the logical equivalent (and AWS only lets us define rate rules per 5 mins)
![Screenshot 2023-04-05 at 15 57 49](https://user-images.githubusercontent.com/75476610/230120895-158c1a22-49c3-4865-a199-806baf5fe7dc.png)

Before when using rack attack we also had some rules for specific endpoints having stricter rate limiting. I added the line "some higher security endpoints have stricter rate limiting" to address this, but not be overly specific (don't want to list those specific endpoints since we change them occasionally)